helm/prometheus-rules/templates/shared/alerting-rules/node.management_cluster.rules.yml

```diff
@@ -26,9 +26,9 @@ spec:
         description: '{{`Node {{ $labels.node }} status is flapping under load.`}}'
       expr: |
         (
-          sum(node_load15{cluster_type="management_cluster", app!="vault", role!="bastion"})
+          sum(node_load15{cluster_type="management_cluster", service="node-exporter"})
```

Comment:
Are you sure you want to be alerted for bastions as well?

Reply:
I've checked on anteater, for the bastions the service label does not have a value; could it be different for other MCs? (if so, which are the minimal test cases? Ideally I want to avoid checking every single one for label disparities :see_no_evil: )

Reply:
I think it's fine :)